src/Limiters/vertical_mass_borrowing_limiter.jl

```diff
@@ -0,0 +1,140 @@
+import .DataLayouts as DL
+
+"""
+    VerticalMassBorrowingLimiter(f::Fields.Field, q_min)
+
+A vertical-only mass borrowing limiter.
+
+The mass borrower borrows tracer mass from an adjacent, lower layer.
+It conserves the total tracer mass and can avoid negative tracers.
+
+At level k, it will first borrow the mass from the layer k+1 (lower level).
+If the mass is not sufficient in layer k+1, it will borrow mass from
+layer k+2. The borrower will proceed this process until the bottom layer.
+If the tracer mass in the bottom layer goes negative, it will repeat the
+process from the bottom to the top. In this way, the borrower works for
+any shape of mass profiles.
```

Comment:
From the reference paper this is true as long as the "the
column-integrated mass is positive" ; We may not need assertions in the code for performance considerations but this is useful in the docstring.